tests/checkasm/vvc_deblock.c

```diff
@@ -72,59 +72,96 @@ static void randomize_params(int32_t beta[4], int32_t tc[4], const int size)
     }
 }
 
-static void randomize_chroma_buffers(int type, int beta[4], int32_t tc[4],
-   uint8_t *buf, ptrdiff_t xstride, ptrdiff_t ystride, const int shift, const int bit_depth)
+static void randomize_chroma_buffers(const int bit_depth, const int type, const int shift, const int spatial_strong,
+uint8_t* max_len_p, uint8_t* max_len_q, int beta[4], int32_t tc[4], 
+uint8_t *buf0, uint8_t *buf1, ptrdiff_t xstride, ptrdiff_t ystride)
 {
     const int size = shift ? 4 : 2;
     const int end = 8 / size;
 
+    switch (type) {
+        case 0: // strong
+            for(int i = 0; i < 4; ++i) {
+                max_len_p[i] = 3;
```

Comment:
the size of max_len_p is not always 4, if shift is zero, max_len_{p, q} is an array with 2 components. 
so you can move this to L188